plugins/modules/swim_intent.py

```diff
@@ -1099,12 +1116,13 @@ def get_diff_distribution(self):
 
         if len(device_uuid_list) == 0:
             self.status = "failed"
-            msg = "No devices found for Image Distribution"
-            self.result['msg'] = msg
-            self.log(msg)
+            self.msg = "No devices found for Image Distribution"
+            self.result['msg'] = self.msg
+            self.log(self.msg, "WARNING")
             return self
 
-        self.log("List of device UUID's for Image Distribution " + str(device_uuid_list))
+        self.log("List of device UUID's for Image Distribution: {0}".format(str(device_uuid_list)), "INFO")
```

Comment:
`    self.log("Device UUIDs involved in Image Distribution: {0}".format(str(device_uuid_list)), "INFO")
`

Reply:
added.